tests/eas/acceptance_small_task_packing.config

```diff
@@ -1,42 +0,0 @@
-{
```

Comment:
This chunk belongs to this path:
tests/eas/acceptance: SmallTaskPacking: Use Python dict not JSON